wgsl/index.bs

```diff
@@ -386,11 +386,31 @@ and to encourage interoperability among tools.
 WGSL program text consists of a sequence of Unicode [=code points=], grouped into contiguous non-empty sets forming:
 * [=comments=]
 * [=tokens=]
-* [=blankspace=]
+* [=blankspaces=]
```

Comment:
nit: It's subtle, but blankspace already is defined as a sequence of those things. It didn't have to be that way but it is.
No need to worry about this.

Reply:
Thanks!